File: spring-cloud-square-demo/consumer-retrofit-okhttp/src/main/java/com/livk/consumer/RetrofitOkhttpConsumerApp.java
package com.livk.consumer;

import com.livk.common.LivkSpring;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * <p>
 * RetrofitOkhttpConsumerApp
 * </p>
 *
 * @author livk
 * @date 2022/4/13
 */
@SpringBootApplication
public class RetrofitOkhttpConsumerApp {

	public static void main(String[] args) {
		LivkSpring.run(RetrofitOkhttpConsumerApp.class, args);
	}

}

File: spring-cloud-square/consumer-retrofit-okhttp/src/main/kotlin/com/livk/consumer/RetrofitOkhttpConsumerApp.kt
package com.livk.consumer

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * @author livk
 */
@SpringBootApplication
open class RetrofitOkhttpConsumerApp

fun main(args: Array<String>) {
    runApplication<RetrofitOkhttpConsumerApp>(*args)
}
